app/code/Magento/QuoteGraphQl/Model/Resolver/SetShippingMethodsOnCart.php

```diff
@@ -57,36 +57,39 @@ public function __construct(
      */
     public function resolve(Field $field, $context, ResolveInfo $info, array $value = null, array $args = null)
     {
-        $shippingMethods = $this->arrayManager->get('input/shipping_methods', $args);
```

Comment:
Pls, verify that is proper changes

Reply:
Yes, this change is required. Please, take a look at the schema change https://github.com/magento/graphql-ce/pull/342/files#diff-795a33fde881f18aba5165a5a8c7513fR63
We have changed the input in order to reflect the new schema from here https://github.com/magento/graphql-ce/wiki/Checkout-workflow

```graphql
mutation {
  setShippingMethodOnCart (
    input: {
      cart_id: "vpLKDPvz0F2P2J3ksx97ZcSHUA95a8PX"
      shipping_addresses: [ <!-- Here is the mentioned change
        {
             ...
```
Thanks.